pkg/amazon/backend/convert.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/docker/ecs-plugin/pkg/compose"
 )
 
+const secretsInitContainerImage = "ndeloof/ecs-secrets"
```

Comment:
this needs to be a docker image

Reply:
Definitively. But as I can't deploy this under `docker` hub account I had to use my own for demo purpose.